pkgs/development/python-modules/asyncpg/aaeb707.patch

```diff
@@ -0,0 +1,37 @@
+From aaeb7076e5acb045880b46155014c0640624797e Mon Sep 17 00:00:00 2001
```

Comment:
I know this is a small patch but is there any way to get it from an external source?

Reply:
Now retrieved from https://github.com/MagicStack/asyncpg/commit/aaeb7076e5acb045880b46155014c0640624797e.patch.